tests/Hedgehog.CSharp.Tests/Hedgehog.CSharp.Tests/LinqTests.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Xunit;
+
+// Import Check and ForAll:
+using static Hedgehog.Property;
```

Comment:
Wow, I've been out of the C# loop for a while, it's awesome that you can do this now 💯